packages/protocol/src/Chief.sol

```diff
@@ -97,12 +100,32 @@ contract Chief is CoreRoles, AccessControl, IChief {
     if (!openVaultFactory && !hasRole(DEPLOYER_ROLE, msg.sender)) {
       revert Chief__deployVault_missingRole(msg.sender, DEPLOYER_ROLE);
     }
+    _checkRatingValue(rating);
+
     vault = IVaultFactory(_factory).deployVault(_deployData);
     vaultSafetyRating[vault] = rating;
     _vaults.push(vault);
+
     emit DeployVault(_factory, vault, _deployData);
   }
 
+  /**
+   * @notice Set `vaultSafetyRating` for `_vault`.
+   * Requirements:
+   *  - Emits a `SafetyRatingChange` event.
+   *  - Only timelock can change rating.
+   *  - `newRating` is in range [1=100].
+   *  - `vault_` is not zero address.
+   */
+  function setSafetyRating(address vault_, uint256 newRating) external onlyTimelock {
```

Comment:
In "deployVault" above, the naming is " underscore + name" while here it becomes "name + underscore"